src/components/WindowSideBarCollectionPanel.js

```diff
@@ -0,0 +1,184 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Typography from '@material-ui/core/Typography';
+import Skeleton from '@material-ui/lab/Skeleton';
+import ArrowUpwardIcon from '@material-ui/icons/ArrowUpwardSharp';
+import CompanionWindow from '../containers/CompanionWindow';
+import IIIFThumbnail from '../containers/IIIFThumbnail';
+
+/** */
+export class WindowSideBarCollectionPanel extends Component {
+  /** */
+  static getUseableLabel(resource, index) {
+    return (resource
+      && resource.getLabel
+      && resource.getLabel().length > 0)
+      ? resource.getLabel().map(label => label.value)[0]
+      : resource.id;
+  }
+
+  /** */
+  isMultipart() {
+    const { collection } = this.props;
+
+    if (!collection) return false;
+
+    const behaviors = collection.getProperty('behavior');
+
+    if (Array.isArray(behaviors)) return collection.includes('multi-part');
+
+    return behaviors === 'multi-part';
+  }
+
+  /** */
+  render() {
+    const {
+      canvasNavigation,
+      classes,
+      collectionPath,
+      collection,
+      id,
+      isFetching,
+      manifestId,
+      parentCollection,
+      updateCompanionWindow,
+      updateWindow,
+      t,
+      variant,
+      windowId,
+    } = this.props;
+
+    /** */
+    const Item = ({ manifest, ...otherProps }) => (
+      <ListItem
+        className={classes.listItem}
+        alignItems="flex-start"
+        button
+        component="li"
+        selected={manifestId === manifest.id}
+        {...otherProps}
+      >
+        { variant === 'thumbnail' && (
+          <ListItemIcon>
+            <IIIFThumbnail
+              resource={manifest}
+              maxHeight={canvasNavigation.height}
+              maxWidth={canvasNavigation.width}
+            />
+          </ListItemIcon>
+        )}
+        <ListItemText>{WindowSideBarCollectionPanel.getUseableLabel(manifest)}</ListItemText>
+      </ListItem>
+    );
+
+    return (
+      <CompanionWindow
+        title={t(this.isMultipart() ? 'multipartCollection' : 'collection')}
+        windowId={windowId}
+        id={id}
+        titleControls={(
+          <>
+            { parentCollection && (
+              <List>
```

Comment:
Should this be using a `MenuList` for keyboard arrow navigability or is the desired behavior to tab through the collection items?

Reply:
I think we are ok here for the `List` because this is the top level thing to go up to the parentCollection.

Reply:
@cbeer is the consideration here for a `List` also how the text should be displayed?

Reply:
I think that's right -- it's visual styling only. And, for now, it's a list of a single item (presumably items that belong to multiple collections could happen in the future), so internal navigation may not matter much here... down on line 189 is a different story -- I suspect this code predated the `MenuList`-ification and probably could/should be updated?

Reply:
Great, I've added a commit that uses the `Menu` approach for consistency.